src/components/MovieDetails/Component/MovieDetails.tsx

```diff
@@ -0,0 +1,36 @@
+import { Movie } from "../../Types/movie";
+import styles from "../Styles/MovieDetails.module.css";
+
+interface Props {
+  movie: Movie;
+}
+
+export default function MovieDetails({ movie }: Props) {
+  const {
+    poster_path,
+    title,
+    release_date,
+    vote_average,
+    runtime,
+    overview,
+    genres,
+  } = movie;
+
+  return (
+    <div className={styles.movieDetails}>
+      <div className={styles.poster}>
+        <img src={poster_path} alt={title} />
+      </div>
+      <div className={styles.info}>
+        <h2>{title}</h2>
+        <span className={styles.rating}>{vote_average}</span>
+        <p className={styles.genres}>{genres?.join(", ")}</p>
+        <p>
+          <span className={styles.release_date}>{release_date}</span>
+          <span className={styles.duration}>{runtime}</span>
```

Comment:
Runtime recieved as a number needs to be converted in the human readable format: 
![Screenshot 2024-03-05 185824](https://github.com/keber23/react-learn/assets/79011727/9a2c4bc0-67e4-42bc-9730-dc0901c04930)